src/v2/guide/syntax.md

```diff
@@ -66,6 +66,8 @@ So far we've only been binding to simple property keys in our templates. But Vue
 {{ message.split('').reverse().join('') }}
 
 <div v-bind:id="'list-' + id"></div>
+
+<div v-bind:style="`background-image: url('${imageUrl}')`"></div>
```

Comment:
Oh, I see! I hadn't seen vue-loader's extensive documentation. Maybe this change is unnecessary in that case.